src/material/button/button.spec.ts

```diff
@@ -349,6 +364,45 @@ describe('MDC-based MatButton', () => {
       buttonNativeElements.every(element => !!element.querySelector('.mat-mdc-focus-indicator')),
     ).toBe(true);
   });
+
+  describe('interactive disabled buttons', () => {
```

Comment:
I don't think we have a styleguide for stuff like this. The unit tests we already have cover the `disabledInteractive = false` case so I just tried to cover the differences here. Also in general I like asserting before and after, because we reuse the fixture components with other tests and sometimes tests pass purely because of their initial state.